.pre-commit-config.yaml

```diff
@@ -81,6 +81,33 @@ repos:
             ^python/cudf/cudf/core/dtypes.py|
             ^python/cudf/cudf/tests/pytest.ini
           )
+      - id: cython-docstrings
+        name: cython-docstrings
+        description: 'Enforce multi-line Cython docstrings start on line two'
+        # We want to allow """One line docstring"""
+        # But disallow
+        # """This is the first line
+        #    Here is the second line"""
+        # Since in REPL help, the indentation is wrong.
+        # It should instead be
```

Comment:
This guidance is equivalent to pydocstyle rule D213 and we should mention that in the description.